src/core.js

```diff
@@ -1024,45 +1022,49 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     };
   }
 
-  function validateChanges(changes, source, callback) {
-    var waitingForValidator = new ValidatorsQueue();
-    waitingForValidator.onQueueEmpty = resolve;
-
-    for (var i = changes.length - 1; i >= 0; i--) {
-      if (changes[i] === null) {
-        changes.splice(i, 1);
-      } else {
-        var row = changes[i][0];
-        var col = datamap.propToCol(changes[i][1]);
+  /**
+   * Get parsed number from numeric string.
+   *
+   * @param {String} numericData Float (separated by a dot or a comma) or integer.
+   * @returns {Number} Number if we get data in parsable format, not changed value otherwise.
+   */
+  function getParsedNumber(numericData) {
+    // Unifying "float like" string. Change from value with comma determiner to value with dot determiner,
+    // for example from `450,65` to `450.65`.
+    const unifiedNumericData = numericData.replace(',', '.');
```

Comment:
How this will work with thousands separator? For example in our system clients can choose out of two combinations:
-  `.` - decimal separator; `,` - thousands separator;
-  `,` - decimal separator; `.` - thousands separator;

Reply:
Property `numericFormat` won't have influence on way you enter data. Client will be able to write floats with both dot and comma separator (without thousands separator). 

You can use custom editor if you need connection between input data basing and `numericFormat` property: https://github.com/handsontable/handsontable/issues/4396#issuecomment-357202908.

Reply:
:ok: